services/base-service/models/middleware/index.js

```diff
@@ -286,6 +286,14 @@ function json_only(req, res, next) {
   }
 }
 
+function public_or_is_user(req, res, next) {
```

Comment:
@UcDust , regarding your question, yes, the server has that same config, and we will replace our guards in the api with this.  So you can turn is_public=false for stage eg.